src/DevChatter.Bot.Infra.Twitch/TwitchChatClient.cs

```diff
@@ -104,9 +109,11 @@ public void SendMessage(string message)
 
         public IList<ChatUser> GetAllChatters()
         {
+            _logger.LogInformation("Getting All Twitch Chatters");
             List<ChatterFormatted> chatters = _twitchApi.Undocumented.GetChattersAsync(_settings.TwitchChannel)
                 .TryGetResult(10).Result;
             var chatUsers = chatters.Select(x => x.ToChatUser()).ToList();
+            _logger.LogInformation("Returning All Twitch Chatters");
```

Comment:
This string, along with "Twitch Client Disconnected", "Getting All Twitch Chatters", and "Twitch Client Connected" use title casing. Is there a reason for this? the other logging strings use sentence casing.

Reply:
Nope. It just happens when you're streaming. Some little things like casing get missed.